### PR TITLE
More detailed README.md

### PR DESCRIPTION
**Pull request by francoiskarman**
Based on @ryanbnl idea.

This readme update provides a better initial statement (as this repo is not to be considered as the reference) and the list of the key valuesets.

### CODE REVIEW

**Review by ryanbnl — APPROVED** [review]

**Review by ryanbnl — APPROVED** [review]

**Review by ryanbnl — APPROVED** [review]
Just saw this, looks good!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+
+
+Name | Description
+-----| ------------------
+test-manf | List of recognised manufactorers of tests. Snapshot provided for development purposes, please refer to the gateway for the actual version.
+test-result | List of supported test results (i.e. detected, not detected..)
+test-type | List of supported test types (i.e. PCR, rapid-antigen..)
+disease-agent-targeted | List of diseases/targeted agent 
+country-2-codes | List of ISO-3166-2 country codes
+
```
Comment (dslmeinte) [inline README.md:13]: Shouldn't the `vaccine-*.json` files also not be in this table?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+This repository contains a snapshot of the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema). The most up-to-date version of the valuesets are distributed on the DCCG (DCC Gateway).
+
+
+Name | Description
+-----| ------------------
+test-manf | List of recognised manufactorers of tests. Snapshot provided for development purposes, please refer to the gateway for the actual version.
+test-result | List of supported test results (i.e. detected, not detected..)
+test-type | List of supported test types (i.e. PCR, rapid-antigen..)
+disease-agent-targeted | List of diseases/targeted agent 
+country-2-codes | List of ISO-3166-2 country codes
```
Comment (dslmeinte) [inline README.md:12]: "plus whatever code we deemed prudent to add" ;)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+
 
 Release: 2.9.0
```
Comment (dslmeinte) [inline README.md:15]: For completeness' sake: the addition of this information should be documented in `CHANGELOG.md` as well.

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (skounis) [reply README.md:13]: I agree with adding all the rest files in the table. It helps understand where the visitor will find the information.

We could merger these changes and improve this table in the next step.

File: README.md (hunk 3 above)
Comment (skounis) [reply README.md:15]: I would support taking the version out of the readme. We have this information already in the "Releases" and the `CHANGELOG.md` 

We could put here links to these two locations

**Review by skounis — CHANGES_REQUESTED** [review]
As per my single comments.

**Comment by dslmeinte** [comment]
We want this in the 2.11.0-release, right? @francoiskarman : can you process the review comments (and possibly merge from the `release/2.11.0` release branch as well)? I can process my own as well, if that's OK.

**Comment by ryanbnl** [comment]
Closing as we rolled this up into #68

**Pull request closed without merging**